weave-js/src/components/Panel2/ChildPanelExportReport/utils.ts

```diff
@@ -3,23 +3,31 @@ import {uriFromNode, determineURISource} from '../../PagePanelComponents/util';
 import {ChildPanelFullConfig} from '../ChildPanel';
 import {format, getYear} from 'date-fns';
 
-export const CREATE_NEW_REPORT_OPTION = 'Create new report';
+export const NEW_REPORT_OPTION = 'New report';
 export const DEFAULT_REPORT_OPTION = {
```

Comment:
**suggestion (non-blocking):** a little confusing that these are both named  `[...]_OPTION` when they're not the same shape

```suggestion
export const NEW_REPORT_OPTION_LABEL = 'New report';
export const DEFAULT_REPORT_OPTION = {
```

Reply:
done done